packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -564,11 +566,17 @@ export async function popupMessage(
   versionForMigration: VersionForMigration,
   isModal: boolean
 ): Promise<string | undefined> {
+  let buttons: string[];
+  if (!isModal) {
+    buttons = [upgradeButton, learnMoreText];
+  } else {
+    buttons = migrationMessageButtons;
```

Comment:
Minor: seems like `migrationMessageButtons` are also reversed, it's for modal dialog and the default value is "learn more".

Reply:
Will discuss with PM and use another PR  to update it if necessary